pysat/tests/classes/cls_instrument_access.py

```diff
@@ -0,0 +1,840 @@
+"""Tests for data access in the pysat Instrument object and methods.
```

Comment:
There are also some tests for data loading, equality method comparisons, index tests, concat, empty, and variable renaming.